src/Bicep.Core/Emit/ResourceDependencyVisitor.cs

```diff
@@ -11,89 +11,42 @@ namespace Bicep.Core.Emit
 {
     public class ResourceDependencyVisitor : SyntaxVisitor
```

Comment:
Seems like you're no longer using this class anywhere. If so, please could you revert the changes to this file?